rules/linux/privilege_escalation_find_binary.toml

```diff
@@ -0,0 +1,68 @@
+[metadata]
+creation_date = "2022/02/28"
+maturity = "production"
+updated_date = "2022/02/28"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies Linux binary find abuse to break out from restricted environments by spawning an interactive system shell'"
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via find Shell Evasion"
+references = ["https://gtfobins.github.io/gtfobins/find/"]
+risk_score = 47
+rule_id = "6f683345-bb10-47a7-86a7-71e9c24fb358"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation","GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id, process.pid with maxspan=1m
+[process where process.name == "find" and process.args : "-exec" and process.args : ("*bash", "*sh") and process.args : ";"]
+[process where process.parent.name == "find" and process.name : ("bash", "sh")]
+'''
+
+
+[[rule.threat]]
```

Comment:
https://attack.mitre.org/techniques/T1127/ & T1059.004 (PrivEsc is not valid in this context, its more about bypass app-whitelisting to run Unix shell)